.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -7,30 +7,41 @@ assignees: comcmaho, amdeel, davidmrdavid, bachuv
 
 ---
 
-**Prep Release: (assigned to:)**
-_Due: <3-business-days-before-release>_
-- [ ] Merge all features and fixes into the `dev` branch
-- [ ] Publish updated versions of DurableTask.Core and DurableTask.AzureStorage to app-service MyGet feed
-- [ ] Update dependencies and increment extension version in the `dev` branch.
-- [ ] Publish signed version of Microsoft.Azure.WebJobs.Extensions.DurableTask to app-service MyGet feed
-- [ ] Close out or punt remaining GitHub issues for the current milestone
+**Prep DTFx Release: (assigned to:)**
+_Due: <2-3-business-days-before-release>_
+- [ ] Increment updated DTFx packages (either DT-AzureStorage only or DT-AzureStorage, DT-Core, and DT-Redis)
+- [ ] Delete DTFx test packages from myget.
+- [ ] Update OneBranch durabletask repo, run signing pipeline (using these notes:  [Durable Task Framework Build Pipeline] (https://microsoft.sharepoint.com/teams/AzureWebjobs/_layouts/OneNote.aspx?id=%2Fteams%2FAzureWebjobs%2FSiteAssets%2FAzureWebjobs%20Notebook&wd=target%28Planning%2FFunctions%20Post-GA.one%7CA43CF112-7272-481A-B23E-9AA5CA8EEE06%2FDurable%20Task%20Framework%20Build%20Pipeline%7CD0946823-6FB0-44E3-A57F-E252617B69CD%2F%29
+onenote:https://microsoft.sharepoint.com/teams/AzureWebjobs/SiteAssets/AzureWebjobs%20Notebook/Planning/Functions%20Post-GA.one#Durable%20Task%20Framework%20Build%20Pipeline&section-id={A43CF112-7272-481A-B23E-9AA5CA8EEE06}&page-id={D0946823-6FB0-44E3-A57F-E252617B69CD}&end) ).
+- [ ] Publish DTFx packages to myget for testing. (either DT-AzureStorage only or DT-AzureStorage, DT-Core, DT-Redis, and DT-Emulator)
 
-
-**Validation (assigned to: )**
+**Prep Release (assigned to: )**
 _Due: <2-business-days-before-release>_
-- [ ] Run private performance tests and ensure no regressions
-- [ ] Smoke test Functions V1 and Functions V3 .NET apps
-- [ ] Smoke test JavaScript and Python apps
-- [ ] Check for package size, make sure it's not surprisingly heavier than a previous release
+- [ ] Update Durable Functions references (Analyzer? DTFx?) and increment version.
+- [ ] Add Durable Functions package to myget staging feed.
+- [ ] Check for package size, make sure it's not surprisingly heavier than a previous release.
+- [ ] Merge dev into main (Merge NOT Squash merge).
+
+**Validation**
+_Due: <1-business-days-before-release>_
+- [ ] Run private performance tests and ensure no regressions. **(assigned to: )**
+- [ ] Smoke test Functions V1, Functions V2, and Functions V3 .NET apps. **(assigned to: )**
+- [ ] Smoke test JavaScript and Python apps. **(assigned to: )**
 
+**DTFx Release Completion (assigned to: )**
+_Due: <release-deadline>_
+- [ ] Publish DTFx packages to nuget (directly to nuget.org).
+- [ ] Publish release notes for DTFx.
 
-** Release Completion (assigned to: )**:
+**Release Completion (assigned to: )**
```

Comment:
I grouped tasks together, but as you can see, there are more than 3/4 roles. Instead of assigning one person each role, we can assign ourselves as much as we're comfortable taking. Some might be on call or have as especially busy after-servicing week (or be OOF), while others might be more free.

Also, we will be not always have a DTFx release accompanying our Durable Functions extension release.

Reply:
I personally still have a preference for 3-4 roles that we rotate between. It doesn't matter if the roles are perfectly balanced in terms of work, as a consistent rotation should even things out over time.

While it is true we may have Durable extension releases without a DTFx release, that is fairly rare.

Reply:
I understand having 3-4 roles makes things easy to assign, but we could accomplish the same thing by assigning the first two to the same person and the second two to the same person if you'd rather we do it that way. I just kept the breakdown in categories to make it easy to see which parts are left out when not doing a DTFx release and just easy to read in general.